packages/server/src/validator/utils.ts

```diff
@@ -1,10 +1,31 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 import { Validator } from '.';
 import { JsonRpcError, predefined } from '@hashgraph/json-rpc-relay';
+import { IObjectSchema, IObjectValidation } from '../types/validator/objectTypes';
+import { IMethodParamSchema } from '../types/validator/methods';
+import { ITypeValidation } from '../types/validator/types';
```

Comment:
if we decide to have multiple files for the validator folder, we should include these new types into an index.ts file at the root of the `types` folder so the import statements can be a little cleaner

Reply:
I agree but maybe not on the root level, but rather on `types/validator` in order to have better separation in the imports

Reply:
yeah that's fair